.github/workflows/h100_lychee.yml

```diff
@@ -0,0 +1,90 @@
+name: Reusable H100 workflow
+
+permissions:
+  contents: none
+
+on:
+  workflow_call:
+    inputs:
+      kokkos_version:
+        description: 'The Kokkos Core version to build'
+        default: ''
+        required: true
+        type: string
+
+jobs:
+  PR_LYCHEE_HOPPER90_CUDA1180_CUDA_LEFT_RIGHT_REL:
+    name: PR_LYCHEE_HOPPER90_CUDA1180_CUDA_LEFT_RIGHT_REL
+    runs-on: [lychee-kk-env-cuda-11.8.0-gcc-11.3.0-latest]
+    
+    steps:
+      - name: checkout_kokkos_kernels
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          path: kokkos-kernels
+
+      - name: checkout_kokkos
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          repository: kokkos/kokkos
+          ref: ${{ inputs.kokkos_version }}
+          path: kokkos
+
+      - name: configure_kokkos
+        run: |
+          mkdir -p kokkos/{build,install}
+          nvidia-smi
+          cd kokkos/build
+          cmake -DCMAKE_CXX_COMPILER=$PWD/../bin/nvcc_wrapper \
+            -DCMAKE_CXX_FLAGS= \
+            -DCMAKE_EXE_LINKER_FLAGS= \
+            -DCMAKE_INSTALL_PREFIX=$PWD/../install \
+            -DKokkos_ENABLE_CUDA=ON \
+            -DKokkos_ARCH_HOPPER90=ON \
+            -DKokkos_ENABLE_TESTS=OFF \
+            -DKokkos_ENABLE_EXAMPLES=OFF \
+            -DCMAKE_VERBOSE_MAKEFILE=ON \
+            -DCMAKE_CXX_EXTENSIONS=OFF \
+            -DBUILD_SHARED_LIBS=OFF \
+            -DKokkos_ENABLE_DEPRECATION_WARNINGS=OFF \
+            -DKokkos_ENABLE_DEPRECATED_CODE_4=OFF \
+            -DKokkos_ENABLE_DEPRECATION_WARNINGS=OFF \
+            ..
+
+      - name: build_and_install_kokkos
+        working-directory: kokkos/build
+        run: make -j56 install
+
+      - name: configure_kokkos_kernels
+        run: |
+          mkdir -p kokkos-kernels/{build,install}
+          cd kokkos-kernels/build
+          cmake \
+            -DCMAKE_CXX_COMPILER=$PWD/../../kokkos/bin/nvcc_wrapper \
+            -DKokkos_DIR=$PWD/../../kokkos/install/lib64/cmake/Kokkos \
```

Comment:
Use `-D Kokkos_ROOT` instead to avoid the `lib` vs `lib64` issue and just make the path simpler.